db/migrate/20231004215656_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.string :comments_counter
+      t.string :likes_counter
+      # t.integer :author_id
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused code in order to have a clean code base.